modules/nodes-nodes-garbage-collection-images.adoc

```diff
@@ -7,8 +7,9 @@
 [id="nodes-nodes-garbage-collection-images_{context}"]
 = Understanding how images are removed through garbage collection
 
-Image garbage collection relies on disk usage as reported by *cAdvisor* on the
-node to decide which images to remove from the node.
+Image garbage collection removes images not referenced by any running pods.
+
+The process relies on disk usage as reported by *cAdvisor* on the node to decide which images to remove from the node.
```

Comment:
This sentence is bugging me a bit (anthropomorphic + IMO, a bit confusing). But! I just realized that you're only removing the hard wrap, so no worries for this PR.